tests/ServerTest.php

```diff
@@ -10,17 +10,29 @@ class ServerTest extends \PHPUnit_Framework_TestCase
     public function testStartStop()
     {
         $factory = new ServerFactory(realpath(__DIR__ . '/..'));
+
+        if (file_exists(BASE_PATH . '/vendor/silverstripe/framework/tests/behat/serve-bootstrap.php')) {
+            // SS4
+            $bootstrapFile = 'vendor/silverstripe/framework/tests/behat/serve-bootstrap.php';
+        } else {
+            // SS3
```

Comment:
You set the requirements to >4.0, so how could this still support ss3?

Reply:
As discussed on Slack. we agree there should be separate release lines for 3.x and 4.x. Somebody has already created the 1.x branch for this repo recently, so I've sent a PR to restrict that release line compat to 3.x: https://github.com/silverstripe/silverstripe-serve/pull/17